source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -1272,26 +1305,33 @@ void ShaderGraph::populateColorTransformMap(ColorManagementSystemPtr colorManage
             // If we're converting between two identical color spaces than we have no work to do.
             if (sourceColorSpace != targetColorSpace)
             {
-                ColorSpaceTransform transform(sourceColorSpace, targetColorSpace, shaderPort->getType());
-                if (colorManagementSystem->supportsTransform(transform))
-                {
-                    if (asInput)
+                // Cache colorspace on shader port
+                shaderPort->setColorspace(sourceColorSpace);
+                shaderPort->setPath(input->getNamePath());
```

Comment:
Is setting the path here intentional? Seems like a surprising side-effect in this method.

Reply:
Hmm. Think I copied this over and is unneeded. Thanks for catching this..